service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/GatekeeperEmailContentProvider.java

```diff
@@ -30,4 +32,16 @@ protected GatekeeperEmailContentProvider(@Value("${ccd.ui.base.url}") String uiB
             .put("localAuthority", config.getLocalAuthorityName(localAuthorityCode))
             .build();
     }
+
+    public String buildRecipientsLabel(List<String> emailList, String recipientEmail) {
+        List<String> filteredEmailList = emailList.stream()
+            .filter(email -> !recipientEmail.equals(email))
+            .collect(Collectors.toList());
```

Comment:
Instead of collecting to an intermediary list, thoughts on collecting to a string straight away? Instead of `Collectors.toList()` you can use `Collectors.joining(", ")` which will still return an empty string if nothing is present in the filtered list